libs/neo4j/langchain_neo4j/vectorstores/neo4j_vector.py

```diff
@@ -93,7 +93,7 @@ def _get_search_index_query(
                 "YIELD node, score "
             ),
             SearchType.HYBRID: (
-                "CALL { "
+                "CALL () { "
```

Comment:
Are we supporting only neo4j >= 5.23 ?

Reply:
Good point, I should include the old queries for older versions

Reply:
https://github.com/langchain-ai/langchain-neo4j/pull/6/commits/ed09e73c1b955a3ea3be47fe90438ad0185abf70

Reply:
I created another ticket to add support for Neo4j < 5.23 across the package. Feel free to add in support here but ideally we should think about exactly which versions we want to support and come up with a strategy that works across the package.